CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-## Next Release
+## v0.4.0 (2022-10-04)
```

Comment:
Newline between markdown headings and their content please. Markdown doesn't like content right up against a header.